README.md

```diff
@@ -70,8 +70,11 @@ route:
   # The labels by which incoming alerts are grouped together. For example,
   # multiple alerts coming in for cluster=A and alertname=LatencyHigh would
   # be batched into a single group.
+  # To aggregate by all labels use '...' instead of label names. 
```

Comment:
There should be an explicit warning that this will create a separate alert for each unique label set, which is only useful in very specific scenarios.